src/app/components/IconItem.jsx

```diff
@@ -4,16 +4,21 @@ const IconItem = ({
   styles = "",
   iconStyles = "",
   textStyles = "",
+  textStyles2 = "",
```

Comment:
The names used for props are not good, we should define what type of text is it going to take. Like if a text is a title then we should define it as a title, and for a paragraph, paragraph, or description.